updating landing page for reference (#969)
elastic/docs-content: reference/index.md

# Reference [api-reference]

Explore the reference documentation for the Elastic Stack. Whether you are configuring security protocols, enhancing search functionalities, or integrating with cloud services, our detailed manuals, API guides, and configuration settings provide the information you need.

## Security

Access detailed reference documentation on field and object schemas as well as the diffferent commands used to manage and troubleshoot Elastic Endpoint.

**Learn more in [Security](security/index.md)**

## Observability

Access detailed reference documentation on field and object schemas as well as the Elastic Entity Model.

**Learn more in [Observability](observability/index.md)**

## Search

Explore reference content on Search UI and how you can develop fast, modern, and engaging search experiences.

**Learn more in [Search](search/index.md)**

> ## Elaticsearch and index management

Reviewer: ```suggestion
## Elasticsearch and index management
```

> Customize your Elastic Stack setup with our configuration reference guides. From JVM settings, our client documentation, Elasticsearch privileges and index lifecycle actions, find the detailed information you need to configure your environment to meet your specific needs.

Reviewer 2: ```suggestion
Customize your Elastic Stack setup with our configuration reference guides. Explore JVM settings, client documentation, Elasticsearch privileges, and index lifecycle actions to find the detailed information you need to configure your environment to your specific needs.
```

**Learn more in [Elasticsearch and index management](elasticsearch://reference/index.md)**

## Ingestion tools

Streamline data ingestion with tools like Fleet and Elastic Agent, APM, and Beats. Explore processor references and Logstash plugins to efficiently manage your data flow.

**Learn more in [Ingestion tools](ingestion-tools/index.md)**

> 

Reviewer: Will these entries be links?  Sending users back to Ingestion tools and forcing them to navigate back to their product of choice doesn't seem right. 

Also, Fleet and Agent are in the bulleted list and also in the side bar. Can we put Logstash in the side bar with entries for Logstash Reference, Plugins docs, and Versioned Plugin docs? 

<img width="1029" alt="Screenshot 2025-04-01 at 5 48 09 PM" src="https://github.com/user-attachments/assets/3b2f5681-220d-458e-9053-134a49aef9c7" />

Author: I am going based off of what is in staging which is different than the preview. In staging those aren't high level items. https://staging-website.elastic.co/docs/reference/

## Kibana

Visualize and analyze your data with Kibana. Configure advanced settings, explore plugins, and utilize command line tools to enhance your data insights.

**Learn more in [Kibana](kibana://reference/index.md)**

## Plugins

Extend the functionality of your Elastic Stack with a variety of plugins. From analysis and discovery to snapshot/restore and store plugins, customize your setup to fit your requirements.

**Learn more in [Plugins](elasticsearch://reference/elasticsearch-plugins/index.md)**

## Query Languages

Master data querying with our comprehensive guides on QueryDSL, ES|QL, SQL, EQL, and Kibana Query Language. 

**Learn more in [Query languages](elasticsearch://reference/query-languages/index.md)**

## Scripting Languages

> Access syntax references, function libraries, and best practices to of Painless scripting.

Reviewer 2: ```suggestion
Access syntax references, function libraries, and best practices for Painless scripting.
```
